### DISCUSSION TITLE
Project Snippet (start with pre-defined packages)

### DISCUSSION THREAD

**Discussion by nephi-dev** [python-poetry/poetry#6181] [Ideas]
There's any way to create snippets to poetry? Like creating a snippet with some pre-defined packages, for example a snippet with the name `default-libs` that when I use `poetry new my_package`  I can pass the arg. `--snippet default-libs` to start the project with pre-defined packages or even pre-defined folders?

**Comment by Secrus** [comment]
Poetry by itself does not offer such functionality. We might consider it as a feature for the future. For now, I can personally recommend [`cookiecutter` project](https://github.com/cookiecutter/cookiecutter).

**Reply by nephi-dev** [reply]
Thanks for the reply, I think this feature can be very helpful for poetry users.
